File: javascript/b1012.js
var input = require("fs").readFileSync("stdin", "utf8");

var values = input.split(" ");

var A = parseFloat(values.shift());
var B = parseFloat(values.shift());
var C = parseFloat(values.shift());

var PI = 3.14159;

var triangle = A * C / 2;
var circle = Math.pow(C, 2) * PI;
var trapezium = (A + B) * C / 2;
var square = B * B;
var rectangle = A * B;

console.log("TRIANGULO: " + triangle.toFixed(3));
console.log("CIRCULO: " + circle.toFixed(3));
console.log("TRAPEZIO: " + trapezium.toFixed(3));
console.log("QUADRADO: " + square.toFixed(3));
console.log("RETANGULO: " + rectangle.toFixed(3));

File: python/b1012.py
A, B, C = map(float, input().split(" "));
areaTri = A * C / 2;
areaCir = 3.14159 * pow(C, 2);
areaTra = (A + B) * C / 2;
areaSqu = B * B;
areaRec = A * B;

print(f"TRIANGULO: {areaTri:.3f}");
print(f"CIRCULO: {areaCir:.3f}");
print(f"TRAPEZIO: {areaTra:.3f}");
print(f"QUADRADO: {areaSqu:.3f}");
print(f"RETANGULO: {areaRec:.3f}");